align/pnr/toplevel.py

```diff
@@ -303,10 +305,12 @@ def place( *, DB, opath, fpath, numLayout, effort, idx, lambda_coeff, select_in_
 
     hyper = PnR.PlacerHyperparameters()
     # Defaults; change (and uncomment) as required
-    hyper.T_INT = 0.75
-    hyper.ALPHA = 0.99925
-    hyper.T_MIN = hyper.T_INT*(hyper.ALPHA**1e4)    # 10k iterations
-    hyper.SEED = 0  # 0 do not override, >0 overwrite the seed
+    hyper.T_INT = 0.5  # Increase for denormalized decision criteria
+    hyper.T_MIN = 0.05
+    hyper.ALPHA = math.exp(math.log(hyper.T_MIN/hyper.T_INT)/1e4)
+    # hyper.T_MIN = hyper.T_INT*(hyper.ALPHA**1e4)    # 10k iterations
+    # hyper.ALPHA = 0.99925
+    hyper.SEED = seed  # 0 do not override, >0 overwrite the seed
```

Comment:
Is the comment correct? Should we have a conditional around this? Do we want to assert that `seed` is not zero if this is being handled somewhere else?

Reply:
So the comment is describing what the C++ code is doing, not what the Python code is doing. This was confusing to me. Maybe adding "C++ code will" in the comment will help.